catalog/controller/payment/ebanx_plugincheck.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+class ControllerPaymentEbanxPlugincheck extends Controller {
+	const EBANX_VERSION = '2.4.0';
+
+	public function index() {
+		$info_list = array(
+			'mysql'          => $this->getSQLVersion(),
+			'php'            => phpversion(),
+			'Opencart'       => VERSION,
+			'ebanx-opencart' => self::EBANX_VERSION,
+			'configs'        => $this->getConfigs(),
+			'plugins'        => $this->getPlugins(),
+		);
+		echo json_encode($info_list);
+	}
+
+	private function getSQLVersion()	{
+		$query = $this->db->query('SELECT version() AS version');
+		return $query->row['version'];
+	}
+
+	private function getConfigs()	{
+		$configs = array();
+
+		if(!is_null($this->config->get('ebanx_express_mode'))){
```

Comment:
You should create a function to validated if config is null
```suggestion
                if (self::isEbanxExpressMode()) 
```
```
private static function isEbanxExpressMode() {
		    return !is_null($this->config->get('ebanx_express_mode'));
}
```